.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
       - uses: actions/setup-dotnet@v4
         with:
-          dotnet-version: "6.0.x"
+          global-json-file: "global.json"
```

Comment:
just curious why is this necessary? Can we just use:
`- uses: actions/setup-dotnet@v4`